x/auth/types/account.go

```diff
@@ -137,14 +138,12 @@ func (acc BaseAccount) Validate() error {
 	return nil
 }
 
-func (acc BaseAccount) String() string {
-	out, _ := acc.MarshalYAML()
-	return out.(string)
-}
-
 // MarshalYAML returns the YAML representation of an account.
 func (acc BaseAccount) MarshalYAML() (interface{}, error) {
```

Comment:
Should we also remove this method?

Reply:
Yes, this is sort of what https://github.com/cosmos/cosmos-sdk/pull/13850 is aimed to be.